koku/api/report/aws/aws_query_handler.py

```diff
@@ -36,6 +37,43 @@
                   'blended_cost', 'tax_type']
 
 
+def _get_replacement_result(param_res_list, access_list):
+    if ReportQueryHandler.has_wildcard(param_res_list):
+        return access_list
+    intersection = param_res_list & set(access_list)
+    if not intersection:
+        raise PermissionDenied()
+    return list(intersection)
+
+
+def _update_query_parameters(query_parameters, access):
```

Comment:
Since we're using the account alias to limit things, I think adding 

```
models.Index(
                fields=['account_alias'],
                name='summary_account_alias_idx',
            ),
```

to `AWSCostEntryLineItemDailySummary` 's `Meta` list of indexes will be a good idea.